src/test/java/org/folio/rest/impl/ValidatorRegistryTest.java

```diff
@@ -463,18 +444,18 @@ public void shouldReturnBadRequestOnPutWhenNoImplementationReferenceSpecifiedFor
   }
 
   @Test
-  public void shouldReturnNotFoundWhenRuleDoesNotExist(final TestContext context) {
+  public void shouldReturnErrorWhenRuleDoesNotExist(final TestContext context) {
     requestSpecification()
       .header(TENANT_HEADER)
       .body(REGEXP_RULE_ENABLED.toString())
       .when()
       .put(TENANT_RULES_PATH)
       .then()
-      .statusCode(HttpStatus.SC_NOT_FOUND);
+      .statusCode(HttpStatus.SC_BAD_REQUEST);
```

Comment:
The API documentation says that 404 is returned if the rule does not exist: https://s3.amazonaws.com/foliodocs/api/mod-password-validator/validator-registry.html#tenant_rules_put
We don't want a breaking change.

Reply:
The test should use a rule *with* id.

Reply:
Fixed test